src/ISA/ISA.Spreadsheet/CompositeHeader.fs

```diff
@@ -0,0 +1,101 @@
+﻿module ISA.Spreadsheet.CompositeHeader
+
+open ISA
+open FsSpreadsheet
+
+module ActivePattern = 
+
+    open ISA.Regex.ActivePatterns
+
+    let mergeTerms tsr1 tan1 tsr2 tan2 =
+        if tsr1 <> tsr2 then failwithf "TermSourceRef %s and %s do not match" tsr1 tsr2
+        if tan1 <> tan2 then failwithf "TermAccessionNumber %s and %s do not match" tan1 tan2
+        {|TermSourceRef = tsr1; TermAccessionNumber = tan1|}
+
```

Comment:
This is very confusing, not sure when this is used? Only on headers after grouping?

Are you sure you cannot recycle this function more?

https://github.com/nfdi4plants/ISADotNet/blob/8d174a3178e115aa16a8f4538e225626d68a497c/src/ISA/ISA/ArcTypes/CompositeHeader.fs#L122